src/components/glossary/results/SearchResults.js

```diff
@@ -1,36 +1,31 @@
-import React, {Component} from "react";
-import "./SearchResults.css"
+import React, { Component } from 'react';
```

Comment:
Remove o import de Component, por favor
```suggestion
import React from 'react';
```